index.html

```diff
@@ -201,14 +204,15 @@
                     <div class="img-float-hold"><img src="../Agro/images/home-baner-hand-image.png" alt=""></div>
                     <div class="img-float-logo"><img src="../Agro/images/logo.png" alt=""></div>
                     <div class="untitled__slideContent">
-                          <span class="text1">more</span>
-                          <span class="text1">with less</span>
-                          <span class="text2">
+                          <span class="text1" style = "font-family:georgia,garamond,serif;font-size:40px;font-style:italic;">More With Less</span>
+                          <br>
```

Comment:
Not satisfied with the changes. I didn't mean to change the font. You need to change the overall style and needs to make it attractive.